android/src/main/java/ly/count/dart/countly_flutter/CountlyFlutterPlugin.java

```diff
@@ -314,6 +314,9 @@ public void callback(RequestResult downloadResult, String error, boolean fullVal
                         break;
                 }
                 result.success(deviceIDTypeString);
+            } else if ("setID".equals(call.method)) {
+                // Countly.sharedInstance().deviceId().setID();
```

Comment:
this is commented out? So it will be going to add when android published

Reply:
Yes, after I have tested, I will uncomment